### ISSUE TITLE
When I configured batch size 4 on v3-8, it was normal. But when I configured batch size 128 on v3-256, it reported OOM.What is the reason?

### ISSUE THREAD

**Issue by joytianya** [young-geng/EasyLM#26]
When I configured batch size 4 on v3-8, it was normal. But when I configured batch size 128 on v3-256, it reported OOM.
What is the reason?

...
--mp_mesh_dim='-1, 1'
--train_dataset.json_dataset.batch_size=128
...

**Comment by joytianya** [comment]
--initialize_jax_distributed=True \

**Comment by joytianya** [comment]
"For example, if we have 8 accelerators for each host and 32 hosts in total, this gives us a total of 256 accelerators. We can use a 2D mesh of shape (8, 32) to specify that one model is partitioned across 32 accelerators for model parallelism, and each parition has 8 replicas for data parallelism."
Does this sentence mean --mp_mesh_dim='8,32'?

**Comment by young-geng** [comment]
The `--mp_mesh_dim='-1, 1'` indicates that you are using a 3D mesh size of 1x256x1 in a v3-256 configuration. To replicate your v3-8 setting and only expand the data parallel axis, you'll need to specify `--mp_mesh_dim='8,1'`.
